### PR TITLE
224 create animation delay component

### PR DESCRIPTION
**Pull request by osvegn**
## New features
- Create AnimationFrequency component. It stores the delay between two frames in an animation.

### CODE REVIEW

**Comment by osvegn** [comment]
Must add this file to a CMakeLists.txt

File: libs/GraphicECS/SFML/Components/AnimationDelayComponent.hpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+*/
+
+#ifndef ANIMATIONDELAYCOMPONENT_HPP_
+#define ANIMATIONDELAYCOMPONENT_HPP_
+
+#include "Component/Component.hpp"
+
+namespace ecs
+{
+    /// @brief This component allow to know the delay between each frame of an animation.
```
Comment (31Nathan) [inline libs/GraphicECS/SFML/Components/AnimationDelayComponent.hpp:15]: allow

Comment (31Nathan) [reply libs/GraphicECS/SFML/Components/AnimationDelayComponent.hpp:15]: allows

**Review by 31Nathan — CHANGES_REQUESTED** [review]

**Review by LucasTesnier — DISMISSED** [review]

File: libs/GraphicECS/SFML/Components/AnimationFrequencyComponent.hpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#ifndef ANIMATIONFREQUENCYCOMPONENT_HPP_
+#define ANIMATIONFREQUENCYCOMPONENT_HPP_
+
+#include <chrono>
+#include "Component/Component.hpp"
+
+namespace graphicECS::SFML::Components
+{
+    /// @brief This component allows to know the frequency between each frame of an animation.
+    class AnimationFrequencyComponent : public ecs::Component {
```
Comment (ZQUEMA) [inline libs/GraphicECS/SFML/Components/AnimationFrequencyComponent.hpp:17]: Name it as AnimationFrequency

**Review by ZQUEMA — CHANGES_REQUESTED** [review]
One minor change

File: libs/GraphicECS/SFML/Components/AnimationFrequencyComponent.hpp (hunk 2 above)
Comment (osvegn) [reply libs/GraphicECS/SFML/Components/AnimationFrequencyComponent.hpp:17]: It's copplient with other component with others SFML component, so it shoold stay AnimationFrequencyComponent

Comment (ZQUEMA) [reply libs/GraphicECS/SFML/Components/AnimationFrequencyComponent.hpp:17]: Ok, my bad, I didn't see that before.

**Review by ZQUEMA — APPROVED** [review]

**Review by LucasTesnier — APPROVED** [review]

**Review by 31Nathan — APPROVED** [review]

**Review by TomDUVAL-MAHE — APPROVED** [review]

**Pull request merged**